app/views/paths/index.html.erb

```diff
@@ -140,7 +140,7 @@
   <!-- The end of small card -->
 <!-- The third child -->
 <div class="parent-div-3 position-absolute" style="right: 0;">
-    <div> <h1 class="girl-assistant-words"> <strong> HELLO! WHO YOU WANT TO BECOME? </strong></h1> </div>
+    <div> <h1 class="girl-assistant-words"> <strong> Hello <%= current_user.name %>! WHO DO YOU WANT TO BECOME? </strong></h1> </div>
```

Comment:
it should be 

`<strong> **HELLO** <%= current_user.name.**upcase** %>! WHO DO YOU WANT TO BECOME? </strong>`

Reply:
@sabinacuccibar I have changed that and now the name is upcased.